vsts/scripts/tagRunTimeImagesForRelease.sh

```diff
@@ -11,6 +11,14 @@ acrPmeProdRepo="oryxprodmcr"
 sourceBranchName=$BUILD_SOURCEBRANCHNAME
 outFilePmeMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/$acrPmeProdRepo-runtime-images-mcr.txt"
 sourceFile="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-acr.txt"
+debianBusterSourceFile="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-acr.buster.txt"
+debianBullseyeSourceFile="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-acr.bullseye.txt"
+debianBookwormSourceFile="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-acr.bookworm.txt"
+
```

Comment:
At this point is the logic that creates `runtime-images-acr.txt` in the runtime case still useful, if it also produces the Debian-specific versions as well? Could we remove it so that we don't keep a file around that we know is incomplete?

Reply:
Resolved offline -- the `runtime-images-acr.txt` file is no longer created in each job but rather only when we're consolidating all of the Debian-specific `.txt` files, so no lingering files will exist here.